spec/unit/pretty_format_spec.rb

```diff
@@ -7,7 +7,7 @@ def method_missing(*args, &block)
     mock_action_view.send *args, &block
   end
 
-  {String: 'hello', Fixnum: 23, Float: 5.67, Bignum: 10**30,
+  {String: 'hello', Fixnum: 23, Float: 5.67, Bignum: 10**30, Symbol: :foo,
```

Comment:
Space inside { missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.